### ISSUE TITLE
Deepen understanding of A11y

### ISSUE THREAD

**Issue by markgammon** [phetsims/qa#741]

**Comment by KatieWoe** [comment]
This was partially done in a meeting on 12/2/21. A follow up meeting to go over screen reader specifics should be planned for the 2022 year when enough people have time. This meeting will be much smaller in scale.
